tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "lib": ["dom", "es2022"],
 
     "jsx": "react",
-    "moduleResolution": "Node",
+    "moduleResolution": "bundler",
```

Comment:
This change was required by the bringing in of the grid background component in react-grid-layout version 2.1.1. This change required changing (possibly improving) some import declarations in the markdown components.

The same change had to be made to toolbox's tsconfig.ts.

I discussed the change with @amcclain and he agreed that it was the right change.